src/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.cs

```diff
@@ -151,4 +151,20 @@ public interface IDebugUnixShellCommandCallback
         /// <param name="exitCode">The exit code of the command, assuming one was printed.</param>
         void OnExit(string exitCode);
     };
+
+    /// <summary>
+    /// Interface to query unix process architecture information.
+    /// </summary>
+    [ComImport()]
+    [ComVisible(true)]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [Guid("61D4C2C4-7CA8-4472-8446-7B1DF5C9D16D")]
+    public interface IDebugUnixProcess
+    {
+        /// <summary>
+        /// Gets the architecture unix process.
+        /// </summary>
+        /// <returns>the string representation of the process's architecture</returns>
```

Comment:
>architecture [](http://example.com/codeflow?start=64&length=12)

Might want to also add: `The format of this is architecture-specific and will match that returned by `uname -m`. Example: x86_64.`